packages/create-cloudflare/e2e-tests/workers.test.ts

```diff
@@ -1,13 +1,14 @@
 import { join } from "path";
 import { describe, expect, test, afterEach, beforeEach } from "vitest";
+import { frameworkToTest } from "./frameworkToTest";
 import { runC3, testProjectDir } from "./helpers";
 
 /*
 Areas for future improvement:
 - Make these actually e2e by verifying that deployment works
 */
 
-describe("E2E: Workers templates", () => {
+describe.skipIf(frameworkToTest)("E2E: Workers templates", () => {
```

Comment:
this is hard to read as it's not obvious to the reader that we don't have any frameworks that target workers. Maybe add a comment as something like

> Skip all worker tests if we are testing a specific framework as there are no frameworks that deploy to workers

or something along those lines...

Reply:
comment added please have a look 🙂